solarSystem.scss

```diff
@@ -0,0 +1,142 @@
+@media screen and (min-width: 600px) {
+  body {
+    background-image: url("cosmos-1853491_1920.jpg");
+  }
+
+  .planet-card-container {
+    display: none;
+    border: white solid 1px;
+  }
+
+  .title-image-container {
+    position: absolute;
+    width: 95%;
+    z-index: 1;
+    border: #ffffff solid 2px;
+    border-radius: 20px;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    #title-image {
```

Comment:
in relation to prior comment, consider setting width to 100% so that it will shrink with the container